vendor/src/github.com/hashicorp/go-plugin/LICENSE

```diff
@@ -0,0 +1,353 @@
+Mozilla Public License, version 2.0
```

Comment:
I am not asking about license compatibility. I am asking about whether or not MPL is compatible with the goals of a login system. Are we comfortable with forcing implementors of login plugins to have to link against code under a copyleft license?

This is going to require more than a one word answer and a link to blog.